### ISSUE TITLE
Incorrect continuations.

### ISSUE THREAD

**Issue by bukzor** [pausan/cblack#5]
Continuations under a two-column style should be four-column indented, but cblack currently uses two spaces instead.

**Comment by sztomi** [comment]
@pausan @bukzor exactly are the continuations supposed to be four-column under a two-column style? That doesn't look right to me. This change also breaks our formatting in many places, so we'll probably just pin cblack and black for now, but I'd like to understand the rationale.

**Comment by pausan** [comment]
Uhm... Indeed I agree with you @sztomi that under 2-spaces indent style, all lines should be indented using 2 spaces and not 4. I've done some quick tests and I don't see the braces/brackets/... continuations to be reindented to four spaces (and neither two) using latest version of black & cblack, so probably something is broken somewhere, probably due to the latest version of black which is still beta. 

I'm thinking on the following changes moving forward, in order to prevent future issues (and also because this project was meant to be a quick and dirty project to fix my own itch but anybody feeling different feel free to fork & change behaviour as pleased).

1. Pair & pin each cblack version with a black version it should work with (to prevent updates from black to break cblack)
2. Extend test.py and make sure it matches a desired format (e.g test/file.py test/file.py.expected)
3. Update code so indentation levels are always multiple of 2 and go in increments of 2 from previous indentation level (which I agree is the aim of this project for as long as there is no --space or similar option in black)

I must say it might take me a some time (or weeks) to make the time to work on this.

**Comment by pausan** [comment]
It took me a while to find the time, but as promised I've reverted this behaviour, now continuations are two-spaces again.

I've added a test that can be manually executed by running ./test.sh plus I've made the tests run with python 3.6, 3.7, 3.8 and 3.9 upon pull requests and pushes through github actions.

Versioning is now changed as well to match it with the paired black version is referencing as a dependence. This will increase stability and prevent it from breaking in the future.

I've made a release to pypi, so feel free to update your cblack to the latest version.

Thanks everybody.
